File: fix-layout.py
import os
import glob
import re

html_files = glob.glob('*.html')

for file in html_files:
    with open(file, 'r', encoding='utf-8') as f:
        content = f.read()

    # 1. Fix missing </div> in logo-container
    old_logo_html = """                        <span class="logo-text">CleanSubmit</span>
                    <button id="close-sidebar-btn" class="icon-btn" aria-label="Close sidebar">"""
    
    new_logo_html = """                        <span class="logo-text">CleanSubmit</span>
                    </div>
                    <button id="close-sidebar-btn" class="icon-btn" aria-label="Close sidebar">"""
    
    content = content.replace(old_logo_html, new_logo_html)
    
    # 2. Remove avatars in HTML
    # We will remove <img ... avatar ...> and similar things.
    content = re.sub(r'<img[^>]*class="[^"]*avatar[^"]*"[^>]*>', '', content)
    content = re.sub(r'<img[^>]*id="sidebarAvatar"[^>]*>', '', content)
    content = re.sub(r'<img[^>]*class="profile-avatar"[^>]*>', '', content)
    
    with open(file, 'w', encoding='utf-8') as f:
        f.write(content)

print("Fixed layout and removed avatars from HTML.")

File: fix-layout.js
const fs = require('fs');
const glob = require('glob');

const files = glob.sync('*.html');

for (const file of files) {
    let content = fs.readFileSync(file, 'utf-8');

    // 1. Fix missing </div> in logo-container
    const oldLogoHtml = `                        <span class="logo-text">CleanSubmit</span>
                    <button id="close-sidebar-btn" class="icon-btn" aria-label="Close sidebar">`;
    
    const newLogoHtml = `                        <span class="logo-text">CleanSubmit</span>
                    </div>
                    <button id="close-sidebar-btn" class="icon-btn" aria-label="Close sidebar">`;
    
    content = content.replace(oldLogoHtml, newLogoHtml);
    
    // 2. Remove avatars in HTML
    content = content.replace(/<img[^>]*class="[^"]*avatar[^"]*"[^>]*>/g, '');
    content = content.replace(/<img[^>]*id="sidebarAvatar"[^>]*>/g, '');
    content = content.replace(/<img[^>]*class="profile-avatar"[^>]*>/g, '');
    
    fs.writeFileSync(file, content, 'utf-8');
}

console.log("Fixed layout and removed avatars from HTML.");
